test/functional/provider/clipboard_spec.lua

```diff
@@ -752,4 +752,11 @@ describe('clipboard (with fake clipboard.vim)', function()
     expect('some some')
     eq('some', eval('getreg("*")'))
   end)
+
+  it('has correct value for clipboard registers #24257', function ()
+    eval('setreg("*", "")')
+    eval('setreg("+", "waddup")')
+    eq(eval('getreg("*")'), "")
```

Comment:
```suggestion
    eq("", eval('getreg("*")'))
    eq("waddup", eval('getreg("+")'))
```
`eq` has a signature of `(expected, actual, context?)`

Reply:
thank you for catching that! I'm new to writing lua tests